hooks/usePagination.ts

```diff
@@ -17,7 +15,10 @@ export const PaginationProvider = PaginationContext.Provider
 export const usePaginationContext = <T>(): PaginationHandler<T> =>
   useContext(PaginationContext)
 
-export const usePagination = <T>(data: T[]): PaginationHandler<T> => {
+export const usePagination = <T>(
+  data: T[],
+  pageSizes = [5, 10, 25, 50]
```

Comment:
Making these the default page sizes instead of a constant because the designs specify a three-column grid